corehq/toggles.py

```diff
@@ -784,6 +795,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_PRODUCT_PATH,
     [NAMESPACE_DOMAIN],
     0.1,
+    always_disabled=['hsph-betterbirth'],
```

Comment:
@benrudolph I forget, is the namespace in an "item" 'domain' by default, or does this need to be 'domain:hsph-betterbirth' or something? Ignore if this question betrays a lack of understanding of what `item` is set to in the `enabled` method.